project/settings.py

```diff
@@ -199,8 +199,16 @@
             "level": "DEBUG",
             "class": "django_db_logger.db_log_handler.DatabaseLogHandler",
         },
+        "console": {
+            "class": "logging.StreamHandler",
+            "formatter": "simple",
+        },
+    },
+    "loggers": {
+        "db": {"handlers": ["db_log"], "level": "DEBUG"},
```

Comment:
Are you sure we want to put all db logging into the db_log at DEBUG level? This will be quite massive.